src/main/scala/cromwell/engine/backend/local/LocalBackend.scala

```diff
@@ -40,11 +42,19 @@ object LocalBackend {
   def hostExecutionPath(workflowName: String, workflowUuid: WorkflowId): Path =
     Paths.get(CromwellExecutions, workflowName, workflowUuid.id.toString)
 
-  def hostCallPath(workflow: WorkflowDescriptor, callName: String): Path =
-    Paths.get(hostExecutionPath(workflow).toFile.getAbsolutePath, s"call-$callName")
+  def hostCallPath(workflow: WorkflowDescriptor, callName: String, callIndex: ExecutionIndex): Path = {
+   val rootCallPath = Paths.get(hostExecutionPath(workflow).toFile.getAbsolutePath, s"call-$callName")
+    callIndex map { index =>
```

Comment:
I think if a map/getOrElse is going to span three lines anyways you might as well go with a pattern match